modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/AutoComplete/index.scss

```diff
@@ -1,5 +1,4 @@
 .dropdown-menu {
-	max-width: 550px;
```

Comment:
Hi @Artartur, I've found a way to make this change not impact all dropdowns, such as **AutoComplete** of Objects and **Notifications**. As `.dropdown-menu` is a common class of **ClayDropDown**, when we remove the `max-width`, the size becomes to short for the other uses outside Object Admin. As you can see on the attached image:

![Screenshot from 2023-07-26 11-12-13](https://github.com/liferay-objects/liferay-portal/assets/26491796/64944d41-b172-416c-b83a-211a7d18b398)


To solve this we can add a property called `menuElementAttrs` on `<ClayDropdown>` of this file:
> object/object-js-components-web/src/main/resources/META-INF/resources/components/AutoComplete/index.tsx

On line 91 of **index.tsx**:

```
<ClayDropDown
	active={!disabled && active}
	menuElementAttrs={{className: 'autocomplete-expand'}}
	*** existing code **
>
```
Then, we can make this on this file:

> object/object-js-components-web/src/main/resources/META-INF/resources/components/AutoComplete/index.scss


```
.autocomplete-expand {
	max-width: 550px;
	width: 100%;

	
}

.dropdown-menu.date-picker-dropdown-menu {
		max-width: 368px;
}

@media (max-width: 768px) {
	.autocomplete-expand{
		max-width: 505px;
	}
}
```

Please, could you make this change and check if it's working? 
Thanks!

Reply:
I got it, I've changed here and now it's working normally, thx! :tada: